client/telemetry/src/worker.rs

```diff
@@ -77,9 +84,13 @@ impl TelemetryWorker {
 	/// The endpoints must be a list of targets, plus a verbosity level. When you send a message
 	/// to the telemetry, only the targets whose verbosity is higher than the verbosity of the
 	/// message will receive it.
+	///
+	/// It can re-use `Node` (connections) if a `NodePool` is provided. Otherwise new `Node` are
```

Comment:
Could you maybe better explain when new nodes are being created?

Reply:
done